vendor/engines/secure_rooms/app/services/secure_rooms/access_rules/base_rule.rb

```diff
@@ -0,0 +1,20 @@
+module SecureRooms
+
+  module AccessRules
+
+    class BaseRule
+
+      def self.call(user, card_reader, accounts, selected)
+        conditions_verdict = condition(user, card_reader, accounts, selected)
+        conditions_verdict || Verdict.new(:pass)
```

Comment:
this is just here to avoid ending every `Rule` with `Verdict.new(:pass)` since that's more of a <default response>/<requirement for traversing the rules list> within `CheckAccess#authorize`  than a notable piece of the logic within each rule